provider/resources.go

```diff
@@ -177,6 +178,9 @@ func Provider() tfbridge.ProviderInfo {
 						Type:     "string",
 						AltTypes: []tokens.Type{makeType(digitalOceanMod, "Region")},
 					},
+					"urn": {
```

Comment:
This is where we specify naming overrides? This is the piece I wanted to learn! Great.

Reply:
This is exactly it :)